### PR TITLE
grammar nit: `it's` ⇒ `its`

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: spec/latest/index.bs [hunk 1]
```diff
@@ -107,7 +107,7 @@ Also, to prevent CORS-related vulnerabilities each page will see a new instance
 Fingerprinting {#fingerprinting-security}
 --------------
 
-Given that the API describes hardware available to the user and it's capabilities it will inevitably provide additional surface area for fingerprinting. While it's impossible to completely avoid this, steps can be taken to mitigate the issue such as ensuring that device names are reasonably generic and don't contain unique IDs. (For example: "Daydream View" instead of "Daydream View, Pixel XL (Black) - Serial: 1234-56-7890")
+Given that the API describes hardware available to the user and its capabilities it will inevitably provide additional surface area for fingerprinting. While its impossible to completely avoid this, steps can be taken to mitigate the issue such as ensuring that device names are reasonably generic and don't contain unique IDs. (For example: "Daydream View" instead of "Daydream View, Pixel XL (Black) - Serial: 1234-56-7890")
```
Comment (toji) [inline spec/latest/index.bs]: second change in this block is not correct.

**Comment by toji** [comment]
One nit, but otherwise thanks for catching my dumb mistakes. :)

File: spec/latest/index.bs (hunk 1 above)
Comment (cvan) [inline spec/latest/index.bs]: good catch!

**Comment by cvan** [comment]
updated, thanks

**Pull request merged**